lib/winston-logzio.js

```diff
@@ -1,6 +1,10 @@
 const Transport = require('winston-transport');
 const stringifySafe = require('json-stringify-safe');
 const logzioNodejs = require('logzio-nodejs');
+const {
+  LEVEL,
+  MESSAGE,
+} = require('triple-beam');
```

Comment:
y do we need this? seems like a less supported library

Reply:
@AlonMiz 
That's how winston uses it. For example https://github.com/winstonjs/logform/blob/master/logstash.js

Reply:
Also, in case of message field missing in `info` we need to take the splat(message) from `info`